documentation/dev-portal/src/clients/websocket/usage.md

```diff
@@ -1,17 +1,17 @@
 # Using Your Client
 The Nym native client exposes a websocket interface that your code connects to. The **default** websocket port is `1977`, you can override that in the client config if you want.
 
-Once you have a websocket connection, interacting with the client involves piping messages down the socket and listening for incoming messages. 
+Once you have a websocket connection, interacting with the client involves piping messages down the socket and listening for incoming messages.
 
-# Message Requests 
-There are a number of message types that you can send up the websocket as defined [here](https://github.com/nymtech/nym/blob/develop/clients/native/websocket-requests/src/requests.rs):  
+# Message Requests
+There are a number of message types that you can send up the websocket as defined [here](https://github.com/nymtech/nym/blob/develop/clients/native/websocket-requests/src/requests.rs):
 
 ```rust,noplayground
 {{#include ../../../../../clients/native/websocket-requests/src/requests.rs:55:97}}
 ```
 
 ## Getting your own address
-When you start your app, it is best practice to ask the native client to tell you what your own address is (from the generated configuration files - see [here](https://nymtech.net/docs/architecture/addressing-system.md) for more on Nym's addressing scheme). If you are running a service, you need to do this in order to know what address to give others. In a client-side piece of code you can also use this as a test to make sure your websocket connection is running smoothly. To do this, send:
+When you start your app, it is best practice to ask the native client to tell you what your own address is (from the generated configuration files <!--add link -->. If you are running a service, you need to do this in order to know what address to give others. In a client-side piece of code you can also use this as a test to make sure your websocket connection is running smoothly. To do this, send:
```

Comment:
left by accident or a note for future?

Reply:
Note for the future. As the original is gone. I can search where it is now.